src/filelock/_api.py

```diff
@@ -111,6 +111,7 @@ def acquire(
         self,
         timeout: Optional[float] = None,
         poll_interval: float = 0.05,
+        *,
```

Comment:
Why this unrelated change?

Reply:
> Add * to function signature to mark the deprecated argument poll_intervall as keyword only.

I don't think we should do this, that would be a breaking change, please remove that. And add a test for the stack level.

Reply:
> I don't think we should do this, that would be a breaking change.

It would not be a breaking change because the positional argument will be captured by `poll_interval`.

Reply:
> And add a test for the stack level.

`pytest` does not support checking `stacklevel` yet (see pytest-dev/pytest#4343). I'll find a workaround.